akd/src/ecvrf/mod.rs

```diff
@@ -5,16 +5,35 @@
 // License, Version 2.0 found in the LICENSE-APACHE file in the root directory
 // of this source tree.
 
+//! This module contains implementations of a
+//! [verifiable random function](https://en.wikipedia.org/wiki/Verifiable_random_function)
+//! (currently only ECVRF). VRFs can be used in the consensus protocol for leader election
```

Comment:
Do we have plans to use VRFs for leader election? Is there a doc?

Reply:
We don't, that's the same comment from the Diem repo. I'll clean it up since we're not using it for that, it's more informational anyways.